src/main/resources/openapi/schemas/reportUseOverTime.json

```diff
@@ -0,0 +1,34 @@
+{
+  "description": "report data entry (agreement line information)",
+  "type": "object",
+  "properties": {
+    "agreementId": {
+      "type": "string",
+      "description": "agreement identifier (UUID)"
+    },
+    "accessCountTimeSlots": {
+      "description": "List of time slots, for example [\"2020-03\", \"2020-04\", \"2020-05\"]",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "journals": {
+      "type": "object",
+      "description": "Journals report",
+      "properties": {
+        "journals": {
```

Comment:
Why is the top-level `journals` and object containing only a lower-level `journals` field? Do you intend to add further properties to the high-level `journals` object?

Reply:
Yes, totalItemRequests and uniqueItemRequests.

Reply:
Excellent.

Maybe rename one or other of the two `journals` properties to avoid confusion?

Reply:
I've removed the lower level `journals`.